packages/near-membrane-base/src/membrane.ts

```
@@ -4212,82 +4233,15 @@ export function createMembraneMarshall(
-                              }
-                              // eslint-disable-next-line no-empty
-                          } catch {}
-                          // We only check for regexp and array buffers here
-                          // since plain arrays and array buffer views are
-                          // marked as live in the BoundaryProxyHandler
-                          // constructor.
-                          //
-                          // Section 25.1.5.1 get ArrayBuffer.prototype.byteLength
-                          // https://tc39.es/ecma262/#sec-get-regexp.prototype.source
-                          // Step 3: If R does not have an [[OriginalSource]] internal slot, then
-                          //     a. If SameValue(R, %RegExp.prototype%) is true, return "(?:)".
-                          //     b. Otherwise, throw a TypeError exception.
-                          try {
-                              if (ObjectHasOwn(target, 'lastIndex')) {
-                                  ReflectApply(RegExpProtoSourceGetter, target, []);
-                                  return true;
-                              }
-                              // eslint-disable-next-line no-empty
-                          } catch {}
-                          // Section 25.1.5.1 get ArrayBuffer.prototype.byteLength
-                          // https://tc39.es/ecma262/#sec-get-arraybuffer.prototype.bytelength
-                          // Step 2: Perform ? RequireInternalSlot(O, [[ArrayBufferData]]).
-                          try {
-                              if ('byteLength' in target) {
-                                  ReflectApply(ArrayBufferProtoByteLengthGetter!, target, []);
-                                  return true;
-                              }
-                              // eslint-disable-next-line no-empty
-                          } catch {}
-                      }
+                      try {
```

do we really want to swallow this error?

---

Yes. The contract there is a boolean result. If it errors while checking live-ness then it's likely not what we want to consider safe-to-be live.